src/it/test-studio/postbuild.groovy

```diff
@@ -0,0 +1 @@
+assert new File(basedir, "target/studio-registries.json").exists()
```

Comment:
Commit the expected extracted file and make the test compare with it.
Ideally, you will have a way to validate its content at a certain level, until it is validated on the other side on import.

Reply:
I prefer to keep this "real" IT test in https://github.com/nuxeo/nuxeo-studio-maven-plugin/tree/master/src/it instead. WFYT?
If you think that this `postbuild` script is disturbing; I can remove it. For me, having the build passing is enough from that part.

Reply:
Tests made by nuxeo-studio-maven-plugin are not more real than those done here. The perfect place to check the produced content is in the root lib: nuxeo-studio-components-common.
Here, you must test the AAMP part: the fact that a file is produced. I would also test the file content because it validates the combination of versions.
Nevermind.